lib/graphql/queries.js

```diff
@@ -904,6 +906,241 @@ export const getCollectiveSourcePaymentMethodsQuery = gql`
   }
 `;
 
+export const getNewCollectivePageQuery = gql`
```

Comment:
Having the queries in the pages was actually intended. Why moving them here?

Reply:
After updating the public message I need to update Apollo cache to reflect the change. You can see this in file: `components/EditPublicMessagePopup.js` line `199`. As I pointed in a comment there I need to replicate the query I want to update. More on that here: https://github.com/apollographql/apollo-client/issues/3909#issuecomment-522349064.

Moving the query there allows me to use it in `EditPublicMessagePopup.js`.

Apollo is not updating itself after the update because the entity updated in the backend is a `Member` (this is what I am returning) but the entity that needs to be update is a `Contributor` (that is constructed using `Member`).

Reply:
Ok.